atlas-mongodb-mixin/README.md

```diff
@@ -0,0 +1,103 @@
+# MongoDB Atlas Mixin
+The MongoDB Atlas mixin is a set of configurable Grafana dashboards and alerts.
+
+The MongoDB Atlas  mixin contains the following dashboards:
+
+- Atlas cluster overview
+- Atlas performance overview
+- Atlas operations overview
+- Atlas elections overview
+- Atlas sharding overview
+
+and the following alerts:
+
+- AtlasHighNumberOfDeadlocks
+- AtlasHighNumberOfSlowNetworkRequests
+- AtlasDiskSpaceLow
+- AtlasSlowHardwareIO
+- AtlasHighNumberOfTimeoutElections
+
+**Please note:**
+- Some metrics may be reset if the MongoDB Atlas cluster is ever reset.
+- Lock related metrics such as deadlocks and wait count do not report data until the respective event has occurred.
+- The Atlas sharding overview dashboard is controlled by a boolean flag in the config file. See more in the [environment configuration](#environment-configuration) section.
+- Within the Atlas sharding overview dashboard certain panels only relate to `shardsvr` nodes, other panels relate only to `mongos` nodes, and other relate to both nodes.
+
+## Atlas cluster overview
+The Atlas cluster overview dashboard provides an at a glance view into an Atlas cluster. Metrics such as hardware I/O's, network requests, memory usage, hardware disk usage, connections, latency, operations, deadlocks, and inventory tables for the nodes in the cluster. Please note that lock related panels such as deadlocks and wait times will show "no data" until load is generated for their respective metrics.
+
+![First screenshot of the MongoDB Atlas cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/atlas-mongodb/screenshots/atlas-cluster-overview-1.png)
+![Second screenshot of the MongoDB Atlas cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/atlas-mongodb/screenshots/atlas-cluster-overview-2.png)
+
+## Atlas performance overview
```

Comment:
Similar sentiments to brand guidelines I think should apply to dashboard names. It seems this was also a diversion from the spec?